Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoIse.yaml

```diff
@@ -0,0 +1,84 @@
+    let ASIM_GetUsernameType = (username:string) { 
```

Comment:
Please check the helper functions. This implementation is already present.

Reply:
changed to usage of helper function